stories/checkbox/index.js

```diff
@@ -24,6 +25,9 @@ stories.add(
       <Checkbox type="secondary" checked>
         This is a secondary checkbox
       </Checkbox>
+      <Checkbox checked disabled>
+        This is a disabled checkbox
```

Comment:
Should also add the Storybook knob below to control the disabled prop (see lines 39-51)

Reply:
Good point. Thanks. Added [here.](https://github.com/curology/radiance-ui/pull/303/commits/3566deec38ff7f37bfbb1d454476490d45654dd7)